releases/templates/README.md

```diff
@@ -0,0 +1,13 @@
+# Templates
+
+This directory contains the templates a release team member might need
+throughout the release.
+
+## notes
```

Comment:
It might be helpful to link the header with the file

```suggestion
## [notes](releases/templates/notes.txt)
```

Reply:
Looks good!